R/as_gt.R

```diff
@@ -397,3 +360,51 @@ footnote_non_binding <- function(x_alpha, full_alpha) {
     "(", a2, " - ", a3, " = ", a1, ") ", "under the null hypothesis."
   )
 }
+
+footnote_row <- function(x, bound) {
+  # for a vector of "Analysis: N", get a logical vector `i`, in which `TRUE`
+  # indicates the position of the largest `N`
+  a <- x$Analysis
+  r <- "^Analysis: ([0-9]+).*"
+  i <- grepl(r, a)
+  k <- as.numeric(sub(r, '\\1', a[i]))
+  i[i] <- k == max(k)
+  i & x$Bound == bound
+}
+
+gs_design_info <- function(
+    x, title, subtitle, spannersub, footnote, bound, columns, inf_bound,
+    alpha_column = spannersub[2], transform = identity
```

Comment:
Might be worth documenting the argument `transform` since it isn't as straight-forward as the others

Reply:
Absolutely. I should document these new functions. Thanks for the suggestion!